src/main/java/org/example/dtos/CastMemberDTO.java

```diff
@@ -16,4 +19,24 @@ public class CastMemberDTO {
     @JsonProperty("known_for_department")
     private String role;
     private String job;
+
+    public CastMemberDTO(CastMember castMember) {
+        this.id = castMember.getId();
+        this.name = castMember.getName();
+        this.role = castMember.getRole();
+        this.job = castMember.getJob();
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        CastMemberDTO that = (CastMemberDTO) o;
+        return Objects.equals(id, that.id) && Objects.equals(name, that.name) && Objects.equals(role, that.role) && Objects.equals(job, that.job);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(id, name, role, job);
+    }
```

Comment:
Remove the override for equals and hashCode as it is already part of the @Data annotation